playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper.go

```diff
@@ -39,10 +39,22 @@ func Setup(sdk pb.Sdk, code string, pipelineId uuid.UUID, workingDir string) (*f
 		return nil, err
 	}
 
+	// copy necessary files
+	switch sdk {
+	case pb.Sdk_SDK_GO:
```

Comment:
Will we have more cases? If not, I think it is better to replace the switch with if.